### PR TITLE
feat: upgrade dependencies

### PR DESCRIPTION
**Pull request by baabouj**

### CODE REVIEW

**Comment by dai-shi** [comment]
Thanks for working on this. Are you familiar with Cypress?

File: package.json [hunk 1]
```diff
@@ -4,32 +4,32 @@
...
-    "@excalidraw/excalidraw": "0.12.0",
-    "browser-fs-access": "0.31.0",
-    "excalidraw-animate": "0.5.0",
+    "@excalidraw/excalidraw": "0.15.2",
+    "browser-fs-access": "^0.33.1",
+    "excalidraw-animate": "0.6.0",
     "gif.js": "^0.2.0",
     "lodash": "^4.17.21",
-    "nanoid": "3.3.4",
+    "nanoid": "4.0.2",
```
Comment (dai-shi) [inline package.json:12]: I think it's easier to use the old version for now. We could eliminate nanoid and look for an alternative.
```suggestion
    "nanoid": "3.3.4",
```

Comment (baabouj) [reply package.json]: nanoid v4 drop support for CommonJS. looks like that's what caused the error

**Comment by baabouj** [comment]
Hey @dai-shi 👋 
Thanks for the review 😄

**Review by dai-shi — APPROVED** [review]
code looks good.

**Comment by dai-shi** [comment]
https://excalidraw-claymate-bl2bgk808-daishi.vercel.app/ works fine.

Only a tiny issue is the help button at the right bottom hides our buttons.

image

**Comment by dai-shi** [comment]
I think it's fine to merge this PR.
Feel free to improve the UI, if anyone is interested.

**Pull request merged**